system-parachains/asset-hubs/asset-hub-polkadot/src/xcm_config.rs

```diff
@@ -264,6 +264,32 @@ impl Contains<Location> for FellowshipEntities {
 	}
 }
 
+pub struct AmbassadorEntities;
+impl Contains<Location> for AmbassadorEntities {
```

Comment:
This doesn't have anything for the `BodyId`. I guess they have no power outside their Treasury/Salary so maybe it is not important. We just have to make sure to add this if they ever do get some new power or need that origin. Perhaps put it in docs?

Reply:
Right. I cannot come up with a good location for such doc. If some collective plurality gets to manage something in Asset Hub, an engineer will need only to update an ensure origin impl for some pallet. I think an engineer should keep in mind that xcm execution is not for free by default. 
We can also just grant a free xcm execution for all Collectives Parachain pluralities, but I am not sure if that will be true for all future pluralities.